src/main/kotlin/no/nav/syfo/dialogmotesvar/kafka/KafkaDialogmotesvar.kt

```diff
@@ -61,7 +61,7 @@ class KafkaDialogmotesvarConsumer(
             validRecords.forEach { record ->
                 val kDialogmotesvar = record.value()
                 val moteUuid = UUID.fromString(record.key())
-                log.info("Received dialogmotesvar with key/moteuuid : $moteUuid of svar type ${kDialogmotesvar.svarType}")
+                log.info("Received dialogmotesvar with key/moteuuid : $moteUuid of svar type ${kDialogmotesvar.svarType} and tekst ${kDialogmotesvar.svarTekst}")
```

Comment:
Kanskje ikke helt greit å logge denne teksten (kan inneholde sensitiv informasjon)?

Reply:
Nei, det var egentlig bare for testing, fjerner den :+1: